packages/suite/src/storage/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 # 28
 
 -   with advent of connect v9, device.state field has changed. migrated all affected data.
+-   split to `token` property for storing OAuth tokens into `tokens.accessToken` and `tokens.refreshToken`
```

Comment:
please move it to version 29

Reply:
Done in b19f1d28478f2a987585f43cc00f81341797f37c